pkg/devfile/adapters/kubernetes/utils/utils.go

```diff
@@ -157,6 +163,65 @@ func UpdateContainersWithSupervisord(devfileObj devfileParser.DevfileObj, contai
 			// Update the containers array since the array is not a pointer to the container
 			containers[i] = container
 		}
+
+		// Check if the container belongs to a debug command component
+		if debugCommand.Exec != nil && container.Name == debugCommand.Exec.Component {
+			// If the run component container has no entrypoint and arguments, override the entrypoint with supervisord
+			if len(container.Command) == 0 && len(container.Args) == 0 {
```

Comment:
is it possible to abstract out the update supervisord binary and volume mount part, it is mostly same in both?

Reply:
Fixed